### PR TITLE
Create BogdanShemetov.md

### PR DESCRIPTION
**Pull request by BogdanShemetov**
## Describe your project in 1-3 sentences

We will automate the process of titration. It is a device widely used in Chemistry for measuring concentration of acid or base.

## Hack Club Slack handle
Shemetov Bohdan
Heipl

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by bellesea** [comment]
heyy @BogdanShemetov thanks so much for making a PR! we're back on tuesday and will start reviewing PRs in the order that they came on. 

to make the process smoother, please ensure you double checked that your file is properly formatted, you have all the parts you need, and that your plan is detailed enough (from a quick glance, i'd appreciate a little more details) for us to imagine how you're using the components in your budget to build your project :)

and fyi, we're looking for projects where/that:
- you can spend at least 10 days building (from Feb 14th to 23rd)
- you are able to modify or make your own in some ways
- you can learn something new from
- involves working with electronics or software on a lower level than a pre-assembled kit of parts

and, we normally don’t approve projects where you buy consumer components w/o modification like keyboards (buying PCB, keycaps, switches, and assembling it), PCs (buying motherboard, cpu, memory and plugging it together), and kits (eg. closed-sourced consumer drone kits).

our Jan 15 11:59pm ET deadline is for PRs to come in and not for final approval so we'll work with you to either accept this project OR give you a chance to submit another PR. 

if you have any questions, please do not hesitate to ask her or DM me (i'm belle on slack), thank you!

File: BogdanShemetov.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+name: "BogdanShemetov"
+project: "Automatic titration"
+---
+
+# Project Name
```
Comment (bellesea) [inline BogdanShemetov.md:6]:
```suggestion
# Automatic titration
```

File: BogdanShemetov.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Automatic titration
```
Comment (bellesea) [inline BogdanShemetov.md:7]:
```suggestion
```

**Review by bellesea — CHANGES_REQUESTED** [review]
hey @BogdanShemetov woah this is such a cool idea!!

a few things:
- can your friend please also comment on this PR confirming they have done the 4 checklist items?
- are you referencing any sort of guides for this?
- under your plan, can you please expand a little on how this machine would work? eg. how would a "user" say interact with this machine? 
- i saw that you're purchasing wires, do you have what you need to cut it?
- does your budget include shipping and taxes?

thank you!

**Comment by BogdanShemetov** [comment]
> hey @BogdanShemetov woah this is such a cool idea!!
> 
> a few things:
>
> * can your friend please also comment on this PR confirming they have done the 4 checklist items?
> * are you referencing any sort of guides for this?
> The project is based on this guide: https://projecthub.arduino.cc/iterjunior/51d08a30-bbf6-4fb1-89a1-dba09ec5fd56
> * under your plan, can you please expand a little on how this machine would work? eg. how would a "user" say interact with this machine?
>  The interaction will be done through a keypad and a screen, that's all that is needed
> * i saw that you're purchasing wires, do you have what you need to cut it?
>  Yes, we do have scissors an all other tools
> * does your budget include shipping and taxes?
> Yes, taxes are included and (approximate) shipping is mentioned at the end

> thank you!

**Comment by Heipl** [comment]
hey @bellesea I'm the second member of the team with @BogdanShemetov , my nickname in Slack is Heipl and I can confirm that:
- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

**Comment by bellesea** [comment]
@BogdanShemetov thanks for your response! i'm curious how this machine works... so what data does it collect, how is the data processed, what decisions does it make, and how does it execute on said decisions (eg. does it automatically add the right amount of acid?)? 

also, what role does your keypad & screen play?

also, with the guide you linked i noticed that within your budget you don't have a pump, temp sensor, etc.?

thank you sm!

**Comment by BogdanShemetov** [comment]
> @BogdanShemetov thanks for your response! i'm curious how this machine works... so what data does it collect, how is the data processed, what decisions does it make, and how does it execute on said decisions (eg. does it automatically add the right amount of acid?)?
> 
> also, what role does your keypad & screen play?
> 
> also, with the guide you linked i noticed that within your budget you don't have a pump, temp sensor, etc.?

So our titrator collects the volume of the used solution(from the burette) and pH of the second solution. Then it displays a graph of pH and calculates the point of equilibrium. It turns on after the user enters all data for calculation and stops after reaching the point of equilibrium.
thank you!

**Review by bellesea — APPROVED** [review]
@BogdanShemetov this is super cool! 

an awesome member of @hackclub/bank-ops will be reaching out soon with next steps!

approving to move forward but can you please:
- update your plan with the details that you mentioned?
- just to check is your valve compatible with a burette or smth that you have? 

thank you!

**Review by MelSmith104 — COMMENTED** [review]
hey @BogdanShemetov! this sounds like a fascinating project idea. id love to hear more about why you chose this to build :) im dm'ing you on slack right now to chat about next steps!

**Comment by BogdanShemetov** [comment]
> * update your plan with the details that you mentioned?
> * just to check is your valve compatible with a burette or smth that you have?

our plan didn't change)

**Comment by bellesea** [comment]
> > * update your plan with the details that you mentioned?
> > * just to check is your valve compatible with a burette or smth that you have?
> 
> our plan didn't change)

@BogdanShemetov i meant this detail. our goal for making the project plans open sourced is also so they can be looked on and referenced in the future so the more details you can add the better :) thank you!

> So our titrator collects the volume of the used solution(from the burette) and pH of the second solution. Then it displays a graph of pH and calculates the point of equilibrium. It turns on after the user enters all data for calculation and stops after reaching the point of equilibrium.

**Comment by BogdanShemetov** [comment]
> > > * update your plan with the details that you mentioned?
> > > * just to check is your valve compatible with a burette or smth that you have?
> > 
> > 
> > our plan didn't change)
> 
> @BogdanShemetov i meant this detail. our goal for making the project plans open sourced is also so they can be looked on and referenced in the future so the more details you can add the better :) thank you!
> 
> > So our titrator collects the volume of the used solution(from the burette) and pH of the second solution. Then it displays a graph of pH and calculates the point of equilibrium. It turns on after the user enters all data for calculation and stops after reaching the point of equilibrium.

We update our plan.
Thank you

**Pull request merged**